_posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+category: news
+published: false
```

Comment:
if you toggle published to true, you'll be able to view the draft at https://precog.mapzen.com/transitland/www-transit-land/italy-blog-post/news/2016/07/19/expanding-our-collection-of-transit-data-in-italy